src/core_plugins/kbn_vislib_vis_types/public/histogram.js

```diff
@@ -29,6 +29,7 @@ export default function HistogramVisType(Private) {
       modes: ['stacked', 'percentage', 'grouped'],
       editor: histogramTemplate
     },
+    isLegendCountSupported: true,
```

Comment:
While this is definitely the right file for this line to go in, I'm not 100% sure this is the right place in the object being passed to the `VislibVisType` constructor. Maybe `isLegendCountSupported` needs to go in `params` or `params.defaults`? It seems a bit hanging out on its own when other parameters are either in `params` or in `params.defaults`. Maybe try out either of those places and also check with Spencer as a 2nd reviewer?